pkg/storage/ingest/partition_offset_client.go

```diff
@@ -317,8 +308,8 @@ func (p *partitionOffsetClient) fetchTopicPartitionsEndOffsets(ctx context.Conte
 	return list, nil
 }
 
-// listTopicPartitionIDs returns a list of all partition IDs in the topic.
-func (p *partitionOffsetClient) listTopicPartitionIDs(ctx context.Context) ([]int32, error) {
+// ListTopicPartitionIDs returns a list of all partition IDs in the topic.
+func (p *partitionOffsetClient) ListTopicPartitionIDs(ctx context.Context) ([]int32, error) {
```

Comment:
Note to reviewers: this function is not used anymore, but I would prefer to keep it for now, if there are no objections (otherwise I will remove it). It's a generic function that we may find useful in the future.